src/ServiceStack.EventStore/Main/EventStoreFeature.cs

```diff
@@ -49,12 +52,21 @@ public EventStoreFeature(SubscriptionSettings subscriptionSettings, params Assem
             log = LogManager.GetLogger(GetType());
         }
 
+        public EventStoreFeature(ClusterVNode clusterNode, params Assembly[] assembliesWithEvents)
+            : this(new SubscriptionSettings(), assembliesWithEvents)
+        {
+            this.clusterNode = clusterNode;
+        }
+
         public async void Register(IAppHost appHost)
         {
             appSettings = appHost.AppSettings ?? new AppSettings();
 
             var connectionSettings = GetConnectionSettings();
-            var connection = EventStoreConnection.Create(connectionSettings.GetConnectionString());
+
+            var connection = clusterNode != null 
+                                ? EventStoreConnection.Create(connectionSettings.GetConnectionString())
+                                : EmbeddedEventStoreConnection.Create(clusterNode);
```

Comment:
Since the EventStoreFeature needs an object that implements `IEventStoreConnection` it might just be best to provide an overload constructor for the `EventStoreFeature` that accepts a previously-created connection.